paper/paper.md

```diff
@@ -23,34 +23,56 @@ bibliography: paper.bib
+Protein kinases are involved in most aspects of cell life due to their role in signal transduction. Dysregulated kinases can cause severe diseases such as cancer, inflammation, and neurodegeneration, which has made them a frequent target in drug discovery for the last decades [@Cohen:2021].
 The immense research on kinases has led to an increasing amount of kinase resources [@Kooistra:2017].
 Among them is the KLIFS database, which focuses on storing and analyzing structural data on kinases and interacting drugs and other small molecules [@Kanev:2021].
 The OpenCADD-KLIFS Python module offers a convenient integration of the KLIFS data into workflows to facilitate computational kinase research.
 
+[OpenCADD-KLIFS](https://opencadd.readthedocs.io/en/latest/databases_klifs.html) (``opencadd.databases.klifs``) is a part of the [OpenCADD](https://opencadd.readthedocs.io/) package, a collection of Python modules for structural cheminformatics.
+
 # Statement of need
 
+The KLIFS resource [@Kanev:2021] contains information about kinases, structures, ligands, interaction fingerprints, and bioactivities. 
+KLIFS thereby focuses especially on the ATP binding site, defined as a set of 85 residues and aligned across all structures using a multiple sequence alignment (MSA) [@vanLinden:2014].
+Fetching, filtering, and integrating the KLIFS content on a larger scale into Python-based pipelines is currently not straight-forward, especially for users without a background in online queries. Effortless switching between data queries from a _local_ KLIFS download and the _remote_ KLIFS database is not possible.
+
 OpenCADD-KLIFS is aimed at current and future users of the KLIFS database who seek to 
 integrate kinase resources into Python-based research projects.
-This module offers access to KLIFS data [@Kanev:2021] such as information about kinases, structures, ligands, 
-interaction fingerprints, and bioactivities. 
-KLIFS thereby focuses especially on the ATP binding site, defined as a set of 85 residues and aligned across all structures using a multiple sequence alignment (MSA) [@vanLinden:2014].
 With OpenCADD-KLIFS, KLIFS data can be queried either locally from a KLIFS download or remotely from the KLIFS webserver. 
-The presented module provides identical APIs for the remote and local queries for KLIFS data and streamlines all output into 
+The presented module provides identical APIs for the remote and local queries and streamlines all output into 
 standardized Pandas DataFrames [@pandas] to allow for easy and quick downstream data analyses (\autoref{fig:opencadd_klifs_toc}). This Pandas-focused setup is ideal to work with in Jupyter notebooks [@Kluyver:2016]. 
```

Comment:
Just doublechecking if correct: the pandas reference reads like this 'The pandas development team, 2020'?

Reply:
That is their official bibtex entry: https://pandas.pydata.org/about/citing.html